CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * [ENHANCEMENT] Blocks storage: Add `-blocks-storage.azure.http.*`, `-alertmanager-storage.azure.http.*`, and `-ruler-storage.azure.http.*` to configure the Azure storage client. #4581
 * [ENHANCEMENT] Optimise memberlist receive path when used as a backing store for rings with a large number of members. #4601
 * [ENHANCEMENT] Add length and limit to labelNameTooLongError and labelValueTooLongError #4595
+* [ENHANCEMENT] Add validation in chunks Redis client's MSet method to avoid panic. #4522
```

Comment:
How would an ordinary user know if they are affected by this?
I think it's not supposed to be user-impacting; if you were going to get a panic then something was badly wrong inside the Cortex code.

Reply:
make sense I'll remove it.